spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -0,0 +1,81 @@
+from typing import Iterable, Union
+import numpy as np
+from scipy.stats import norm
+from spikeinterface.core import BaseRecording, BaseRecordingSegment, get_chunk_with_margin
+from spikeinterface.core.core_tools import define_function_from_class
+from .basepreprocessor import BasePreprocessor, BasePreprocessorSegment
+
+
+class GaussianFilterRecording(BasePreprocessor):
+    """
+    Class for performing gaussian filtering/smoothing on a recording.
+
+    Parameters
+    ----------
+    recording: BaseRecording
+        The recording extractor to be filtered.
+    band: list[float, float]
+        Band (low, high) in Hz of the bandpass filter.
```

Comment:
Actually one suggestion. Can we have `freq_min` and `freq_max` to unify the API with the other bandpass filter?

Reply:
Done! :)